osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -136,55 +149,48 @@ private void onSuccess(CommentBundle response)
         protected override void OnLoadStarted()
         {
             votesCounter.FadeOut(duration, Easing.OutQuint);
-            updateDisplay();
+            updateDisplay(false);
         }
 
         protected override void OnLoadFinished()
         {
             votesCounter.FadeIn(duration, Easing.OutQuint);
-
-            if (IsHovered)
-                onHoverAction();
+            updateDisplay(IsHovered);
         }
 
         protected override bool OnHover(HoverEvent e)
         {
-            onHoverAction();
+            if (!isOwnComment && !IsLoading)
+                updateDisplay(true);
+
             return base.OnHover(e);
         }
 
         protected override void OnHoverLost(HoverLostEvent e)
         {
-            updateDisplay();
+            if (!isOwnComment && !IsLoading)
+                updateDisplay(false);
```

Comment:
very confused why this got so complicated and spread around now. can't the checks of all these flags be inside `updateDisplay()` to centralise all this display logic?

also, use `Enabled` directly, not `isOwnComment`.

Reply:
`Enabled` is jumping back and forth when loading state is changing, that's why I'm still saving `isOwnComment` value

Reply:
then i'm not sure this was worth it because the flag means two things now. partially means that `LoadingButton` is badly designed but alas. i would have asked about this beforehand if i were you, might be wiser to revert at this point.

Reply:
Also not sure if we want to move these checks into `updateDisplay()`, imo it will become very hard to read with more if/else branches

Reply:
it's already hard to read in the current state. i'm not sure it would be that much worse.